pkg/edivisive.py

```diff
@@ -0,0 +1,78 @@
+"""EDivisive Algorithm from hunter"""
+
+import json
+import pandas as pd
+from hunter.report import Report, ReportType
+from hunter.series import Metric, Series
+from pkg.algorithm import Algorithm
+
+
+class EDivisive(Algorithm):
+    """Implementation of the EDivisive algorithm using hunter
+
+    Args:
+        Algorithm (Algorithm): Inherits
+    """
+    def output_json(self):
+        _, series = self._analyze()
+        change_points_by_metric = series.analyze().change_points
+        dataframe_json = self.dataframe.to_json(orient="records")
+        dataframe_json = json.loads(dataframe_json)
+
+        for index, entry in enumerate(dataframe_json):
```

Comment:
I was just thinking if we can abstract out these structure that give us only a list of changepoints and the full list of json. As they can reused by any model integration. @shashank-boyapally any thoughts?

Reply:
Will be adding this to in follow-on PR

Reply:
Can we convert this comment into an RFE issue? So we don't lose track